pkgs/top-level/lua-packages.nix

```diff
@@ -59,6 +59,46 @@ in
   # a fork of luarocks used to generate nix lua derivations from rockspecs
   luarocks-nix = callPackage ../development/tools/misc/luarocks/luarocks-nix.nix { };
 
+ lua-resty-core = callPackage ({ fetchFromGitHub }: buildLuaPackage rec {
```

Comment:
both are available on luarocks: https://luarocks.org/search?q=lua-resty-core+ https://luarocks.org/modules/avlubimov/lua-resty-lrucache , can we generate them instead please ? makes maintainance easier.

Reply:
Those are unofficial outdated uploads. For lua-resty-lrucache the version on rocks is 0.09 and 0.13 in the repo.

Reply:
you can use rockspecs from the repo too. The issue is that the repo doesn't contain any rockspec. One can pass an alternative rockspec but it would be easier for everyone if https://github.com/openresty/lua-resty-lrucache/pull/42 could be merged. Maybe nudge the maintainers and ask for an update on luarocks.org while at it ?

Reply:
but do we really need one? I have tested the package locally already and it worked.